system/settings/settings.xml

```diff
@@ -49,6 +49,9 @@
         <setting id="videoplayer.usedisplayasclock" type="boolean" label="13510" help="36166">
           <level>1</level>
           <default>false</default>
+          <dependencies>
+            <dependency type="enable" setting="audiooutput.passthrough" operator="is">false</dependency>
```

Comment:
It's not possible to that.

As soon as "Sync to display" is enabled it's not possible to select "Allow passthough" and if "Allow  passthrough" had been previously enabled, it would be possible to select ""Sync to display"